lib/jsdom/web-idl/implementation/EventTarget.js

```diff
@@ -0,0 +1,231 @@
+"use strict";
```

Comment:
Do you think it might be easier to keep files next to their IDL, instead of in a sibling folder?